projects/dsp-ui/src/lib/search/fulltext-search/fulltext-search.component.html

```diff
@@ -105,3 +66,48 @@ <h4 mat-line (click)="doPrevSearch(item)" class="dsp-previous-search-item">
     </div>
 
 </ng-template>
+
+
+<!-- PHONE VERSION -->
+<div class="mobile-only">
+    <!-- search buttton only to open a menu with fulltext-search -->
+    <button mat-button class="dsp-fulltext-search-button" [class.active]="displayPhonePanel"
```

Comment:
> On desktop, the search history obstructs the search input field for me

This happened when the browser window has a too small height? Yes, this could be a problem. Can try to fix it. Usually the search panel is at the top of the page in the header toolbar. But you never know 😉

Reply:
> Is there a reason we don't show the search bar by default on mobile? It's a bit confusing needing to tap on a magnifying glass to open the search bar when this button usually means "perform the search". Also if we show the search bar by default, I don't think there will be a need for a whole separate div for a mobile version. We can just modify it's appearance with css when it hit certain @media breakpoints

It's the setup Flavie created in the app. I just moved it to the ui-lib. We can discuss it. The main problem will be the needed space when we have project filter

Reply:
> > On desktop, the search history obstructs the search input field for me
> 
> This happened when the browser window has a too small height? Yes, this could be a problem. Can try to fix it. Usually the search panel is at the top of the page in the header toolbar. But you never know 😉

Ah yes, that's probably the issue. When I click on the full text button, the search history is placed under the search bar. Maybe we can adjust the height of the search history box? And when it goes below a defined min-height, it can be placed above the search bar.

Reply:
Should be resolved in dbf0579